doc/rst/sw_spdif_rx.rst

```diff
@@ -3,36 +3,40 @@
 S/PDIF Receive
 ==============
 
-XMOS devices can support S/PDIF receive up to 192kHz - see ``lib_spdif`` for full specifications.  
+XMOS devices can support S/PDIF receive up to 192kHz - see ``lib_spdif`` for full specifications.
 
-The S/PDIF receiver module uses a clock-block and a buffered one-bit port. 
-The clock-block is divided of a 100 MHz reference clock. The one bit port is buffered to 4-bits.  
-The receiver code uses this clock to over sample the input data.  
+The S/PDIF receiver module uses a clock-block and a buffered one-bit port.
+The clock-block is divided off a 100 MHz reference clock. The one bit port is buffered to 4-bits.
```

Comment:
this is no longer true, lib_spdif uses a 32-bit buffered port.

Reply:
fixed it to 'buffered to 32 bits'